CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.2.0
+Added `unlockWithUUID` to enable unlocking just with the tile id (UUID format needed)
```

Comment:
Same comment on react one, maybe call it `unlockWithDeviceId`?

Reply:
I can change the parameter name to `deviceId` if that helps?
I'd like to keep now signatures that the Android 2.2.0 has been released.

Otherwise I can release 2.2.1

Reply:
Yeah, i know I'm being pedantic but i think it might be worth calling it deviceId everywhere, I don't want people coming to us confusing the tileId and deviceId concepts